### DISCUSSION TITLE
[3차원그래픽스]챕터2 연습문제 2번 질문

### DISCUSSION THREAD

**Discussion by Lcat12345** [MyeongWoonJang/cpp-study#102] [Q&A] [answered]
문제
---

> 선행대수에서 회전 행렬의 역행렬은 그 전치 행렬과 같다(이는 증명 가능하다.) 두 개의 비표준 직교 정규 기저 {a, b, c}와 {d, e, f}가 주어졌을 때, {a, b, c}에서 정의된 벡터를 {d, e, f}에서 정의된 벡터로 변환하는 3x3 행렬을 계산하라.

Q.
---
기저 변환에 대한 내용인데 
이동을 a-d, b-e, c-f라고 생각했습니다. 이것을 3x3행렬로 표현하였죠. (a-d)x (a-d)y (a-d)z ... 이런식으로

그다음 회전행렬을 곱해야 하는데 이는 기저를 그대로 가져와서 쓴다고 했으니 ax ay az bx by bz ... 이런식의 3x3행렬을 그대로 곱해주면 되는건가요?

+이동의 경우 동차좌표를 써야한다는 것을 고려안했네요

올바른 풀이인지 모르겠어서 질문 남깁니다.

**Comment by Woon-2** [comment, marked as answer]
http://mathurl.com/   
링크를 이용해서 행렬이나 수식을 표현한 뒤 이미지를 캡쳐해서 첨부해보세요.

"수식 입력 사이트"라고 검색했습니다.

<br>

https://github.com/Woon-2/cpp-study/discussions/91 
본문 항등식의 세 번째 항이 답입니다.

**Reply by Woon-2** [reply]
그리고 본문에서 첨자 표현을 할 때에 아래를 참고하세요.

a<sub>x</sub>   
**a**<sup>T</sup>

각각
`a<sub>x</sub>`    
`**a**<sup>T</sup>`   
와 같이 적었습니다.
